js/index.js

```diff
@@ -1,6 +1,51 @@
-// Iteration 1: Names and Input
-//
-// Iteration 2: Conditionals
+console.log("I'm ready!")
 
+console.log("Empieza el 1")
 
-// Iteration 3: Loops
+    const hacker1="antonio"
+    console.log("The driver's name is "+hacker1)
```

Comment:
Está bien la expresión, pero generalmente para interpolar variables se realiza así --> 
`console.log(`The driver's name  is ${hacker1}.` )`